CICD/ExtensionMethods.cs

```diff
@@ -889,6 +889,27 @@ public static async Task<Milestone> CloseMilestone(this IMilestonesClient client
         return updatedMilestone;
     }
 
+    /// <summary>
+    /// Returns a value indicating whether or not the GitHub issue is a standard issue.
+    /// </summary>
+    /// <param name="issue">The GitHub issue to check.</param>
+    /// <returns><c>true</c> if a standard issue.</returns>
+    public static bool IsIssue(this Issue issue) => issue.PullRequest is null;
+
+    /// <summary>
+    /// Returns a value indicating whether or not the GitHub issue is a pull request.
+    /// </summary>
+    /// <param name="issue">The GitHub issue to check.</param>
+    /// <returns><c>true</c> if a standard issue.</returns>
+    public static bool IsPullRequest(this Issue issue) => issue.PullRequest is not null;
+
+    /// <summary>
+    /// Returns a value indicating whether or not a GitHub issue is a release to do issue.
+    /// </summary>
+    /// <param name="issue">The GitHub issue to check.</param>
+    /// <param name="releaseType">The type of release issue.</param>
+    /// <returns><c>true</c> if a release pull request.</returns>
```

Comment:
Refactor to below throughout the code base
```suggestion
    /// <returns><c>true</c> if it's a release pull request.</returns>
```